src/libs/filters/rating.c

```diff
@@ -99,21 +99,81 @@ static gchar *_rating_print_func(const double value, const gboolean detailled)
         return g_strdup(_("rejected"));
       case 0:
         return g_strdup(_("not rated"));
-      case 1:
-        return g_strdup("★");
-      case 2:
-        return g_strdup("★ ★");
-      case 3:
-        return g_strdup("★ ★ ★");
-      case 4:
-        return g_strdup("★ ★ ★ ★");
-      case 5:
-        return g_strdup("★ ★ ★ ★ ★");
     }
   }
   return g_strdup_printf("%.0lf", floor(value));
 }
 
+static gchar *dtgtk_range_select_get_rating_bounds_pretty(GtkDarktableRangeSelect *range)
```

Comment:
maybe rename it `_rating_get_bounds_pretty` to be consistent with the other functions in the file...